subsys/bluetooth/host/conn.c

```diff
@@ -339,10 +339,12 @@ void bt_conn_reset_rx_state(struct bt_conn *conn)
 	conn->rx = NULL;
 }
 
-static void bt_acl_recv(struct bt_conn *conn, struct net_buf *buf,
-			uint8_t flags)
+static void bt_acl_recv_(struct bt_conn *conn, struct net_buf *buf, uint8_t flags)
```

Comment:
Was the trailing `_` intended? 
If so, why?

Reply:
yes it _was_ wrapped below. That's not the case anymore, I changed strategies.